src/client/src/app/site/[site]/view-availability/week/view-week-availability-page.tsx

```diff
@@ -0,0 +1,38 @@
+import { Card, Table } from '@components/nhsuk-frontend';
+import { DailyAvailability } from '@types';
+import dayjs from 'dayjs';
+
+type Props = {
+  availability: DailyAvailability[];
+};
+
+export const ViewWeekAvailabilityPage = ({ availability }: Props) => {
+  return (
+    <>
+      {availability.map((day, i) => (
+        <Card title={dayjs(day.date).format('dddd D MMMM')} key={i}>
+          <Table
+            headers={['Time', 'Services', 'Booked', 'Unbooked', 'Action']}
+            rows={day.sessions.map(session => {
+              return [
+                `${session.from} - ${session.until}`,
+                `${session.services.join(',')}`,
+                '0 booked',
+                '0 unbooked',
+                'Change', // TODO: This should be a link
```

Comment:
I did exactly the same thing when first creating the Create Availability page which had a table with a column of Change links, and Vince quite rightly asked me to remove it until we do the ticket that implements it. Sorry to be a pain but I think we should remove this "Action" column for now :)